src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -750,6 +750,15 @@ public void update() {
         actions.put(Actions.REMOVE_FROM_GROUP, new GroupAddRemoveDialog(this, false, false));
         actions.put(Actions.MOVE_TO_GROUP, new GroupAddRemoveDialog(this, true, true));
 
+        /**
```

Comment:
Remove your author comments. 
We keep our authors in a separate file: https://github.com/JabRef/jabref/blob/master/AUTHORS